dmutils/api_stubs.py

```diff
@@ -139,7 +139,7 @@ def brief(status="draft",
             "title": "I need a thing to do a thing",
             "frameworkSlug": framework_slug,
             "frameworkName": framework_name,
-            "frameworkFramework": framework_framework,
+            "frameworkFramework": framework_family,
```

Comment:
Our api stub for `Brief` is out of date compared with our `Brief.serialize()` method. As well as the top level `frameworkFoo` keys (which are due to be deprecated), we also have a nested `framework` dict with `family, name, slug, status`: https://github.com/alphagov/digitalmarketplace-api/blob/master/app/models/main.py#L1574.  

For bonus fun, spot my hilarious typo in that TODO comment ... 🤦‍♀️